molecule_libvirt/playbooks/create.yml

```diff
@@ -15,7 +15,7 @@
     libvirt_default_image_path: "{{ user_home_directory}}/.local/molecule/{{ molecule_scenario_name }}/vms"
     libvirt_vm_arch: "x86_64"
     libvirt_uri: "qemu:///system"
-    default_image_url: "https://mirror.csclub.uwaterloo.ca/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
+    image_url: "https://mirror.csclub.uwaterloo.ca/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
```

Comment:
Please switch to use a CDN enable url instead like one taken from: https://alt.fedoraproject.org/cloud/
 - i think something like https://download.fedoraproject.org/pub/fedora/linux/releases/34/Cloud/x86_64/images/Fedora-Cloud-Base-34-1.2.x86_64.qcow2 is far better for general purpose use.

Reply:
done. @ssbarnea do you know if there is a generic url that point to the latest fedora qcow2 image ?

Reply:
> done. @ssbarnea do you know if there is a generic url that point to the latest fedora qcow2 image ?

https://download.fedoraproject.org/pub/fedora/linux/releases/34/Cloud/x86_64/images/Fedora-Cloud-Base-34-1.2.x86_64.qcow2

Reply:
I mean something like : https://download.fedoraproject.org/pub/fedora/linux/releases/latest/Cloud/x86_64/images/Fedora-Cloud-Base-latest.x86_64.qcow2 .. that will redirect to https://download.fedoraproject.org/pub/fedora/linux/releases/34/Cloud/x86_64/images/Fedora-Cloud-Base-34-1.2.x86_64.qcow2

Reply:
I've got it.

> > @ssbarnea the issue is fixed now but I've got a problem with `libvirt.pc`. It should be present in the host but it has to be installed with yum/dnf or apt.
> > > apt-get install libvirt-dev
> > > or
> > > yum install libvirt-devel
> > 
> > 
> > Since the CI nodes don't have this library installed (I don't know how to have this library installed on the nodes, any help will be welcomed), so the pull request tests will failed.
> > @goldyfruit if you can help, don't hesitate.
> 
> When I removed `libvirt-python` in setup.cfg and CI checks are successfull.

But now, I have to install manually ``libvirt-python`` before running ``molecule test``.